core_ui/lib/screens/file_picker_screen/file_picker_screen.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:convert';
```

Comment:
i have just general question. why did you put all parsing and rendering related files to core_ui package?

Reply:
because i think its not necessary to create one more layer and new packages in our case